### PR TITLE
Document incubator/ro-cert-manager race condition with cert-manager

### PR DESCRIPTION
**Pull request by ivanfetch**
This also documents [this issue](https://github.com/reactiveops/charts/issues/97) in the incubator/ro-cert-manager ReadMe.

### CODE REVIEW

File: incubator/ro-cert-manager/README.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 Installs the necessary additional items that the cert-manager chart lacks.
 
+## Known Issues
+
+### Race Condition With Cert-manager Chart
+
+When this chart is installed too quickly after versions 0.6+ of the cert-manager chart, the cert-manager validating webhook may not yet be initialized, making cert-manager incapable of creating Clusterissuer objects.
+
+Until this chart is updated to address this issue, we recommendseting `cert-manager.enabled` to `false` for this chart, and installing the cert-manager chart separately. Allow one minute for the cert-manager webhook to initialize before installing this chart.
```
Comment (davekonopka) [inline incubator/ro-cert-manager/README.md]: Minor typo here: "recommendseting"

Comment (davekonopka) [reply incubator/ro-cert-manager/README.md]: Another very minor clarification: This starts off saying "Until this chart is updated to address this issue" but I think it means "Until the cert-manager chart is updated".

**Review by davekonopka — APPROVED** [review]
I left two minor comments inline. Otherwise 👍🏻

Thanks for the thorough documentation updates on this here and in docs-engineering.

**Comment by davekonopka** [comment]
@ivanfetch Can you bump the patch version on the chart? I realize it's a documentation change but the linter enforces it.

**Review by endzyme — APPROVED** [review]
I think this looks good (outside of the typo fixes mentioned above)

**Pull request merged**